<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Database_DockWidget</class>
 <widget class="QDockWidget" name="Search_Database_DockWidget">
  <property name="windowTitle">
   <string>Search Database</string>
  </property>
  <widget class="QWidget" name="Search_Database_dockWidgetContents">
   <layout class="QGridLayout" name="Search_Database_dockWidgetContents_gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="Search_toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Fields_page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>319</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Search In Fields</string>
       </attribute>
       <layout class="QGridLayout" name="Fields_page_gridLayout">
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="Search_groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="Search_groupBox_gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="Search_Database_comboBox">
             <property name="statusTip">
              <string>Search Database Combo Box : Database Fields Types.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="Search_Database_lineEdit">
             <property name="statusTip">
              <string>Search Database Line Edit : Searches Input Text Through The Database.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="Search_Database_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QGroupBox" name="Tags_Cloud_groupBox">
             <property name="title">
              <string>Tags Cloud</string>
             </property>
             <layout class="QGridLayout" name="Tags_Cloud_groupBox_gridLayout">
              <item row="0" column="0">
               <widget class="QListWidget" name="Tags_Cloud_listWidget">
                <property name="movement">
                 <enum>QListView::Static</enum>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::IconMode</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="Case_Insensitive_Matching_groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="Case_Insensitive_Matching_groupBox_gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="Case_Insensitive_Matching_label">
             <property name="text">
              <string>Case Insensitive Matching</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="Case_Insensitive_Matching_checkBox">
             <property name="statusTip">
              <string>Case Insensitive Check Box : Database Search Will Ignore Case.</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="Case_Insensitive_Matching_horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>33</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <spacer name="Fields_Page_verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Shot_Time_page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>286</width>
         <height>230</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Search In Shot Time</string>
       </attribute>
       <layout class="QGridLayout" name="Shot_Time_page_gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="Shot_Time_groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="Shot_Time_groupBox_gridLayout">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <item row="3" column="0">
            <widget class="QLabel" name="To_label">
             <property name="text">
              <string>From :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTimeEdit" name="Time_Low_timeEdit">
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="statusTip">
              <string>Time Low Time Edit : Time From When To Search.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="From_label">
             <property name="text">
              <string>To :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QTimeEdit" name="Time_High_timeEdit">
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="statusTip">
              <string>High Time Edit : Time To When To Search.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
             <property name="time">
              <time>
               <hour>23</hour>
               <minute>59</minute>
               <second>0</second>
              </time>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="Shot_Time_page_verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
